### ISSUE TITLE
Exception List

### ISSUE THREAD

**Issue by ubless607** [jayesh-bhoot/enforce-browser-fonts#6]
Can you add an option to always disable addon working on websites that are in the exception list  (addon-setting)?

If the addon is working on websites like google slides, the fonts looks overlapping.

**Comment by bhootd** [comment]
Yes. I intend to introduce a list-like configuration in the settings as soon as time permits.

In the meantime, once you toggle the add-on for a website/domain, it should remember the setting when you visit the website next time. Let me know if that does not happen.
